provers/blevm/script/build.rs

```diff
@@ -1,5 +1,6 @@
 use sp1_helper::build_program_with_args;
 
 fn main() {
-    build_program_with_args("../blevm", Default::default())
+    build_program_with_args("../blevm", Default::default());
+    build_program_with_args("../blevm-mock", Default::default());
```

Comment:
this seems to overwrite the elf binary: `riscv32im-succinct-zkvm-elf`

Reply:
dang, I'll look into how we can persist both ELFs

Reply:
Disabled it in https://github.com/celestiaorg/celestia-zkevm-ibc-demo/pull/129